shared/haml/course_explorer_table.haml

```diff
@@ -20,11 +20,141 @@
-- courses << { name: "CS Fundamentals: Express", starts: 6, ends: 12, subtitle: "Condensed version of curriculum in one 30 hour course for older students", link: CDO.code_org_url("/educate/curriculum/express-course"), img: "/shared/images/courses/logo_tall_express.jpg", description: "Computer Science Fundamentals Express is a 30 hour course designed for 9-18 year old students. It can be taught once a week in an after school club, integrated as a unit in a longer technology class, or used as an intro unit in a computer science class. Students learn to create computer programs that will help them learn to collaborate with others, develop problem-solving skills, and persist through difficult tasks. They will study programming concepts, computational thinking, digital citizenship, and develop interactive games or stories they can share.", prereqs: "No prerequisites needed." }
-- courses << { name: "CS Discoveries", starts: 7, ends: 9, subtitle: "Semester or full year course", link: CDO.code_org_url("/educate/csd"), img: "/shared/images/fit-500/courses/logo_csd.jpg", description: "CS Discoveries is an introductory Computer Science course that empowers students to create authentic artifacts and engage with CS as a medium for creativity, communication, problem solving, and fun. The course will be piloted in Spring 2017, with full rollout Summer 2017.", prereqs: "No prerequisites needed. This course works for beginners or students with experience in CS Fundamentals." }
-- courses << { name: "CS Principles", starts: 9, ends: 12, subtitle: "Full year course", link: CDO.code_org_url("/educate/csp"), img: "/shared/images/teacher-announcement/csp-widgets.png", description: "In fall 2016, the College Board launched its newest AP® course, AP Computer Science Principles. The course introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. The AP Program designed AP Computer Science Principles with the goal of creating leaders in computer science fields and attracting and engaging those who are traditionally underrepresented with essential computing tools and multidisciplinary opportunities. The curriculum is flexible to be taught as a normal course or as an AP course.", prereqs: "No prerequisites needed. This course works for beginners or students with experience in CS Fundamentals or CS Discoveries."}
+:ruby
+  courses = []
+  courses << {
+    name: "CS Principles",
+    starts: 9,
+    ends: 12,
+    subtitle: "Full year course",
+    link: CDO.code_org_url("/educate/csp"),
+    course_link: CDO.studio_url("/courses/csp"),
+    img: "/shared/images/teacher-announcement/csp-widgets.png",
+    description: 'AP Computer Science Principles is a course designed to prepare students (and teachers) who are new to computer science for the AP CS Principles exam. The course covers many topics including the Internet, Big Data and Privacy, and Programming and Algorithms.
```

Comment:
Oh, there is a closing quote a few lines down.  I think there's a better way to declare the value of a multi-line string..